frontend/Panel.js

```diff
@@ -331,9 +339,7 @@ class Panel extends React.Component {
         extraTabs={extraTabs}
         preferencesPanelShown={this.state.preferencesPanelShown}
         theme={theme}
-        onViewElementSource={
-          this.props.showElementSource ? this.viewElementSource.bind(this) : null
-        }
+        onViewElementSource={this.hasLaunchEditor ? this.viewElementSource.bind(this) : null}
```

Comment:
We might not be able to do this anymore, as the global hook could be installed at any time and not cause a re-render (obviously). A lot of children components rely on this, however (to determine if it should add `onClick` and `cursor` changes).

If CRA installs this immediately (before a crash), it might not be a problem.